### PR TITLE
Update Spectral.msg

### PR DESCRIPTION
**Pull request by sampowell-gill**
Add a new intensity unit option for flux

### CODE REVIEW

File: msg/Spectral.msg [hunk 1]
```diff
@@ -25,8 +25,8 @@ uint8 type			# values communicated by spectrum
 
 uint8 intensity_units		# units for collected radiation data
 				# 0 - Counts (detector events)
-				# will almost certainly be counts per channel
```
Comment (martimorta-at-work) [inline msg/Spectral.msg:28]: Should we keep this comment?

Comment (sampowell-gill) [reply msg/Spectral.msg:28]: I don't think so for our case as we will probably use 0 and 1 almost equally.
I took this as the original author suggesting why they haven't defined any other units

Comment (martimorta-at-work) [reply msg/Spectral.msg:28]: It makes sense then

**Review by martimorta-at-work — APPROVED** [review]
Accepting change to be able to use the flux unit. Comment removed due to not being relevant any more.

**Pull request merged**